src/email/templates/orderConfirmation.ejs

```diff
@@ -0,0 +1,166 @@
+
+                    <!-- START MAIN CONTENT AREA -->
+                    <tr>
+                        <td class="wrapper" style="font-family: sans-serif; font-size: 14px; vertical-align: top; box-sizing: border-box; padding: 20px;">
+                            <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;">
+                                <tr>
+                                    <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Hi <%= firstName %>,</p>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Your merch order on the ACM membership portal has been placed:</p>
+                                        <table border="0" cellpadding="0" cellspacing="0" class="btn btn-primary" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; box-sizing: border-box;">
+                                            <tbody>
+                                            <tr>
+                                                <td align="left" style="font-family: sans-serif; font-size: 14px; vertical-align: top; padding-bottom: 15px;">
+                                                    <table border="0" cellpadding="0" cellspacing="0" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: auto;">
+                                                        <tbody>
+                                                        <tr>
+                                                            <th>Item</th>
+                                                            <th>Price</th>
+                                                            <th>Quantity</th>
+                                                            <th>Total</th>
+                                                        </tr>
+
+                                                        <% for(var i=0; i < order.items.length; i++) { %>
+                                                            <tr>
+                                                                <td><%= order.items[i].itemName %></td>
+                                                                <td><%= order.items[i].price %></td>
+                                                                <td><%= order.items[i].quantityRequested %></td>
+                                                                <td><%= order.items[i].total %></td>
+                                                            </tr>
+                                                        <% } %>
```

Comment:
The `price` and `total` don't reflect discounted items, they reflect the actual price of the items even though the discounted price is subtracted from the user's credits. I think you should probably display discounted price here rather than raw price

Reply:
Good catch, will do.